src/site/filters/liquid.js

```diff
@@ -422,19 +422,27 @@ module.exports = function registerFilters() {
     currentPath,
     pageTitle,
   ) => {
-    breadcrumbs.push({
+    const filteredCrumbs = [];
+    breadcrumbs.forEach(crumb => {
```

Comment:
I think you could change the filter code bit to this if you want 🙂 doesn't matter though as i know this is high priority
```js
const filteredCrumbs = breadcrumbs.filter(crumb => crumb.url.path !== '/resources');
```